app/assets/javascripts/spree/frontend/spree-split-payments.js

```diff
@@ -69,11 +70,11 @@ SplitPayments = {
       this.uncheckNonPartialPaymentMethod();
       return true;
     } else {
-      if(!this.non_partial_payment_methods.filter(':checked').length) {
+      if(this.non_partial_payment_methods.filter(':checked').length || this.use_existing_card_checkbox[0].checked) {
```

Comment:
@sachinvinsol @siddharth28 - Are we sure on this that there will always an element in use_existing_card_checkbox?